dev-qt/libqt5pas/libqt5pas-1.2.9.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Pascal wrapper for Qt5"
+DESCRIPTION="libqt5pas is an interface between Lazarus and the Qt5 libraries.
+Not all Qt5 functions are available, only those necessary for Lazarus \
+functionality."
+HOMEPAGE="https://wiki.freepascal.org/Qt5_Interface"
+COPYRIGHT="2005-2013 by Jan Van hijfte"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/davidbannon/libqt5pas/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a40c69f56dcce909417d63d5e1fa8a9f0ddaec9024de6e6f5a99043b99e2f151"
+SOURCE_DIR="libqt5pas-$portVersion/cbindings"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libqt5pas$secondaryArchSuffix = $portVersion
+	lib:libQt5Pas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libqt5Core$secondaryArchSuffix
+	lib:libqt5Gui$secondaryArchSuffix
+	lib:libqt5Network$secondaryArchSuffix
+	lib:libqt5PrintSupport$secondaryArchSuffix
+	lib:libqt5Widgets$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libqt5pas${secondaryArchSuffix}_devel = $portVersion
+	devel:libQt5Pas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libqt5pas$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libqt5Core$secondaryArchSuffix
+	lib:libqt5Gui$secondaryArchSuffix
```

Comment:
Are you sure with lib:?

Reply:
jikes :) strangly enough it didn't error out there ;)

Reply:
Thanks ! Fixed now :)